apps/web/src/components/onboarding/onboarding-wizard.tsx

```diff
@@ -0,0 +1,211 @@
+"use client";
+
+import React, { useCallback, useMemo } from "react";
+import { completeOnboarding } from "@/actions/onboarding";
+import { updateMailInformation, updateUserDataInformation, updateUserGoals } from "@/actions/profile";
+import DataFormFields from "@/components/profile/data-form-fields";
+import MailSettingsFormFields from "@/components/profile/mail-settings-form-fields";
+import UserGoalsFormFields from "@/components/profile/user-goals-form-fields";
+import {
+    createMailSettingsSchemaFromReportSelectType,
+    createUserDataSchemaFromUserDataSelectType,
+} from "@/lib/schema/conversion/profile";
+import { mailSettingsSchema, userDataSchema, userGoalSchema } from "@/lib/schema/profile";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { ArrowRightIcon } from "lucide-react";
+import { useForm } from "react-hook-form";
+import { toast } from "sonner";
+import type { z } from "zod";
+
+import type { ReportSelectType, UserDataSelectType, UserDataType } from "@energyleaf/db/types";
+import { Button, Form, useWizard, Wizard, WizardPage } from "@energyleaf/ui";
+
+interface Props {
+    userData: UserDataType;
+}
+
+export default function OnboardingWizard({ userData }: Props) {
+    function finishHandler() {
+        toast.promise(completeOnboarding(), {
+            loading: "Schließe Onboarding ab...",
+            success: "Onboarding abgeschlossen",
+            error: "Fehler beim Abschließen des Onboardings",
+        });
+    }
+
+    return (
+        <Wizard finishHandler={finishHandler}>
+            <InformationStep />
+            <UserDataStep userData={userData.user_data} />
+            <GoalStep userData={userData.user_data} />
+            <MailSettingsStep reports={userData.reports} />
+            <ThankYouStep />
+        </Wizard>
+    );
+}
+
+function InformationStep() {
+    function onSkip() {
+        toast.promise(completeOnboarding(), {
+            loading: "Überspringe Onboarding...",
+            success: "Onboarding übersprungen",
+            error: "Fehler beim Überspringen des Onboardings",
+        });
+    }
+
+    return (
+        <WizardPage title="Onboarding">
+            <p>
+                Um die App in vollem Umfang nutzen zu können, sollten Sie die in den folgenden Schritten geforderten
+                Daten angeben. Möchten Sie Ihre Daten nicht angeben, so können Sie diesen Schritt auch überspringen und
+                die Daten später in Ihren Profil angeben.
+            </p>
+            <div className="flex w-full justify-center pt-3">
+                <Button
+                    className="flex flex-row items-center justify-center gap-2 text-sm text-muted-foreground"
+                    variant="ghost"
+                    onClick={onSkip}
+                >
+                    Überspringen
```

Comment:
Kann man das Überspringen iwie nach unten in den Dialog schieben oder so?
Ziel ist, dass das nicht so prominent in der Mitte in dem Dialog dargestellt wird, weil wir ja so von der Benutzerführung das Überspringen als einfachste Funktion in die Mitte des Wizards packen. Die Daten sind schon entscheidend für uns

Reply:
Klar sind die Daten wichtig für uns, aber in Kombination mit https://github.com/pgenergy/energyleaf/pull/68#discussion_r1568554846 finde ich das sehr gefährlich bzw. schon fast sehr unethisch. Denn wenn wir im Text nicht explizit darauf hinweisen, dass man das ganze Onboarding überspringen kann, und der Button selbst so versteckt ist, dass man ihn nicht findet, dann forcieren wir die Nutzer letztlich tatsächlich, die Daten anzugeben und das Überspringen würde aus diesem Grund nie genutzt werden, weil es keine reale Option für den Nutzer ist. 
Ich denke, wir sollten hier einen Mittelweg finden. Meiner Ansicht nach ist die Anpassung des Texts im anderen Kommentar besser. Der Button sollte nämlich meiner Meinung nach zumindest in der Card bleiben, da alles außerhalb der Card eine Background-Color erfordern würde und dann zu mehr Aufmerksamkeit führt. Und unter den Step-Buttons finde ich das jetzt auch nicht so schön, da man bei jedem Step mit der Maus in die Nähe von dem Button kommt, sodass man bei jedem Step an das Überspringen erinnert wird. Ich würde auch mal die These aufstellen, dass der Button da in der Mitte weniger auffällt, weil viele Leute die Wizards einfach schnellstmöglich durchklicken wollen (was ja auch ein Grund ist, warum die ganze Freeware, die man sich über Wizards dazu installiert kann (z. B. irgendwelche random Anti-Viren-Programme) dann auch tatsächlich bei Leuten auf dem PC landet.